examples/multisig_transfer.ts

```diff
@@ -0,0 +1,163 @@
+// FIXME: remove this check once the Zones .bind(env) fix is merged
+const hostname = Deno.env.get("TEST_CTX_HOSTNAME");
+const portRaw = Deno.env.get("TEST_CTX_PORT");
+if (!hostname || !portRaw) {
+  throw new Error("Must be running inside a test ctx");
+}
+
+import { createKeyMulti } from "../deps/polkadot/util-crypto.ts";
+import * as C from "../mod.ts";
+import * as T from "../test_util/mod.ts";
+import * as U from "../util/mod.ts";
+
+// Signatories must be sorted
+const signatories: Uint8Array[] = [
+  T.alice.publicKey,
+  T.bob.publicKey,
+  T.charlie.publicKey,
+].sort();
+const THRESHOLD = 2;
+const multisigPublicKey = createKeyMulti(signatories, THRESHOLD);
+
+const env = C.Z.env();
+
+// Transfer initial balance (existential deposit) to multisig address
+U.throwIfError(await addBalanceToMultisigAddress().bind(env)());
+
+// Create proposal
+U.throwIfError(
+  await createOrApproveMultisigProposal({
+    publicKey: T.alice.publicKey,
+    sign: C.Z.call(0, () =>
+      (message) => ({
+        type: "Sr25519",
+        value: T.alice.sign(message),
+      })),
```

Comment:
@harrysolovay I found a weird issue.
If a plain function is used this doesn't work.
It seems that the result of the first `sign()` call is cached so on the 2nd extrinsic (when bob approves the proposal) the signature check fails

Reply:
For now, let's just use the test ctx and call each with `run` (wihout first binding to an env).